src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -27,6 +27,10 @@
 				{ width: 335, viewSize: 375 },
 				{ width: 280 },
 			]"
+			:city="city"
+			:state="state"
+			:country-name="countryName"
+			:distribution-model="distributionModel"
```

Comment:
There's also a world wherein we wrap the borrower-image and the summary tag in a div and do this layout in our card instead of embedding in the BorrowerImage component.

Reply:
I see no strong objections to having the option to activate the Summary within BorrowerImage so let's proceed for now.

Reply:
Is that the prefered route?